test/controllers/api/docs_controller_test.exs

```diff
@@ -58,16 +52,59 @@ defmodule HexWeb.API.DocsControllerTest do
     assert conn.resp_body == "HEYO"
   end
 
+  @tag :integration
```

Comment:
not sure if this tag is needed here, as most integration testing around releasing docs is covered in the previous test